index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Portfolio</title>
+  <link rel="stylesheet" href="style.css">
+  <link href='http://fonts.googleapis.com/css?family=Roboto' rel='stylesheet' type='text/css'>
+</head>
+<body>
+  <div id="blurBg">
+    <header>
+      <a href="#"><h2>John Doe</h2></a>
+      <div class="menuImg"><img src="./images/menu.png" class="menu" alt=""></div>
+
+      <nav>
+        <ul class="menu_items" id="menuList">
+          <li><button type="button" id="btn_close">X</button></li>
+          <li><a href="#project-s">Portfolio</a><span>></span></li>
+          <li><a href="#headline">About</a><span>></span></li>
+          <li><a href="#contact-me">Contact</a><span>></span></li>
+          <li><img src="./images/mail.png" class="mail" alt="image"><span>></span></li>
+        </ul>
+      </nav>
+    </header>  
+  
+    <main>
+      <section id="headline">
+        <article class="headlines">
+          <h1> Hey There. <br> I'm Dennis </h1>
+          <h2> I'm a Software Developer </h2>
+        </article>
```

Comment:
- [ ] `article` doesn't have any semantic meaning here. Kindly remove that.
`You can check` [here](https://www.w3schools.com/tags/tag_article.asp) `when to use the article tag`
`hint: You may use generic div inf you want to`